src/components/common/mediaPicker/results/SourceSearchResultsContainer.js

```diff
@@ -270,19 +321,23 @@ const mapStateToProps = state => ({
     'allMedia',
     'advancedSearchQueryString',
   ),
+  links: state.system.mediaPicker.sourceQueryResults.linkId,
 });
 
 // tags holds metadata search tags
 const mapDispatchToProps = (dispatch, ownProps) => ({
   updateMediaQuerySelection: (values) => {
     if (values && values.tags) {
+      dispatch(resetMediaPickerSources()); // clear prev results
       if (values.allMedia) { // handle the "all media" placeholder selection
-        ownProps.updateMediaQuerySelection({ media_keyword: values.mediaKeyword, type: values.type, allMedia: true });
+        ownProps.updateMediaQuerySelection({ media_keyword: values.mediaKeyword || '*', type: values.type, allMedia: true });
```

Comment:
`'*'` is referenced a few times in this file and I'm actually not sure what used for. Wildcard? It would be clearer if that were a constant, e.g. `WILDCARD = '*'`

Reply:
I think I made a wildcard constant at one point. will find it or make a new one

Reply:
I think you may have meant to use the newly created `WILDCARD_ASTERISK` here